### DISCUSSION TITLE
瑟瑟不好

### DISCUSSION THREAD

**Discussion by aitann** [share121/inter-knot#2192] [其他]
![17234818063442593fb54de6452e3f2a68f58518df142c9a968f082690aa18f253d1fcdbbfb75 0](https://github.com/user-attachments/assets/877b8622-ce02-45d1-acd9-968b2a68d5fa)

**Comment by hlytxdy** [comment]
就要瑟瑟就要瑟瑟

**Comment by NOVAQUQ** [comment]
![Screenshot_2024-03-22-02-37-51-07_149003a2d400f6adb210d7e357a3a646](https://github.com/user-attachments/assets/afc036f0-520c-494a-9da5-c79c353d8b01)
我要瑟瑟_(´ཀ`」 ∠)__
